wsmaster/che-core-sql-schema/src/main/resources/che-schema/5.16.0/1__add_installer.sql

```diff
@@ -0,0 +1,59 @@
+--
+-- Copyright (c) 2012-2017 Codenvy, S.A.
+-- All rights reserved. This program and the accompanying materials
+-- are made available under the terms of the Eclipse Public License v1.0
+-- which accompanies this distribution, and is available at
+-- http://www.eclipse.org/legal/epl-v10.html
+--
+-- Contributors:
+--   Codenvy, S.A. - initial API and implementation
+--
+
+-- Installer ---------------------------------------------------------------------
+CREATE TABLE installer (
+    id                  VARCHAR(255)         NOT NULL,
```

Comment:
Can you make id is auto generated id, and installer_id+version as an alternative unique key. It will allow in nested tables to depend on single field id.